app/src/main/res/layout/main_activity.xml

```diff
@@ -43,6 +43,7 @@
 
         <FrameLayout
             android:id="@+id/main"
+            android:background="@color/splashscreen_background"
```

Comment:
Because I was too lazy to make a new colour, although I could've made a new colour and pointed it to the spash background anyways. The background of subs with a dark theme was still white, now it uses the splash background, which is white by default, and grey on the theme i was testing with